charts/project-origin-wallet/templates/deployment-wallet.yaml

```diff
@@ -98,10 +100,16 @@ spec:
                   name: {{ .Values.persistance.cloudNativePG.name }}-app
                   key: password
             - name: ConnectionStrings__Database
-              value: Host=$(DB_HOST); Port=$(DB_PORT); Database=$(DB_DATABASE); Username=$(DB_USERNAME); Password=$(DB_PASSWORD);
+              value: "Host=$(DB_HOST);Port=$(DB_PORT);Database=$(DB_DATABASE);Username=$(DB_USERNAME);Password=$(DB_PASSWORD);"
+          {{- else if eq .Values.persistance.type "BYOD" }}
+            - name: ConnectionStrings__Database
+              valueFrom:
+                secretKeyRef:
+                  name: {{ required "BYOD is selected as the database type, but no secretName is provided in persistance.byod" .Values.persistance.byod.secretName }}
+                  key: connectionString
```

Comment:
this key...

Reply:
Done. I also fixed the spelling of `persistance`

https://github.com/project-origin/wallet/pull/254/commits/e41f7f37cb79efb22cca56161278040ed49bca16